### ISSUE TITLE
add, respond to and notify of user assertions

### ISSUE THREAD

**Issue by M-Nicholls** [AtlasOfLivingAustralia/DataQuality#106]
Notify user that made the annotation when the record is updated or annotation is replied to

View annotations report for data set owner
Bulk update annotations by data set owner

New annotation types:
•	Duplicate
•	Not a duplicate
•	Verify identification
•	Verify location
•	Verify date

**Comment by M-Nicholls** [comment]
I know this is a much bigger can of worms, but I'm interested to know what you expect an ALA user to do these days if they detect that a record has undetected issues.

I'm starting some work on the plume moth section of the Moths of Victoria series and immediately noted these records from the VP:

https://biocache.ala.org.au/occurrences/bf09820f-0261-4661-a61b-1047362af013 
https://biocache.ala.org.au/occurrences/efc58902-18f3-4b18-b0d5-42014be1373a

They are duplicates that exist for the same basic VP dataset being included twice. I'm not sure why. Maybe it had something to do with me ...

In any case, these two records are placed in Victoria, well outside the range otherwise known for the species. I wouldn't be surprised if the species is in fact found in Victoria, at least near the NSW south coast. However, I strongly suspect that "Stradbrooke" (the only locality information on the label is "Stradbroke Island, QLD" rather than "Stradbroke, VIC". The clincher is the number of other Queensland records from the same period with similar labels, e.g.:

https://biocache.ala.org.au/occurrences/92a48434-98b7-4d80-8371-fb612ac6950f 

So - if I had control over this record, I'd immediately shift the coordinates to an imprecise blob over Stradbroke Island. But I'm not ...

Apparently, the ALA portal still allows me to subscribe for annotations on any dataset but I can't see any way to add any. 

Would contacting the VP team and seeking to change it at their end work? Or is the dataset now crystalised and never reimported in ALA?

This is a small example and I'm not asking anyone to do anything about it. I'm just interested to know if we have any remedies open to someone wanting to fix a problem.

**Comment by pbrenton** [comment]
I don't think that getting the VP to do something about will work as the VP team really only provide support to institutions (such as ANIC) to use the system and the institutions own the data. The VP is just a vehicle to digitisation of the records. Once the expedition (project) is complete and verified, the images are often deleted to make room on the server for new expeditions - data remains (I think) for calculation of the user/usage statistics. 

The issue with the 2 records indicated would most likely be in the verification rigour and associated QA processes and that would be with ANIC in this case. Interestingly the data providers are different and it looks like one record has come direct from ANIC while the other has come from DigiVol direct. It would probably be fair to remove obvious duplicate records sourced directly from DigiVol where they have a corresponding duplicate from an institutional data provider. This would solve part of the problem, then for the record correction you'd go back to the institutional data provider as that would be the true source record. 

I think you'd still need to include the institutional record matching as there will be some data resources sourced directly from DigiVol that won't have been initiated by an institution with a collection system.
